third_party/accessibility/gfx/native_widget_types.h

```diff
@@ -192,7 +192,8 @@ typedef base::android::ScopedJavaGlobalRef<jobject> NativeEvent;
 constexpr NativeView kNullNativeView = nullptr;
 constexpr NativeWindow kNullNativeWindow = nullptr;
 #else
-#error Unknown build environment.
+// for unknown platform.
+typedef void* NativeCursor;
```

Comment:
Why are we not making this #error?

Otherwise we'll potentially crash at runtime?

Reply:
this file will go through linux lint even if it is not part of the build. I fear if we add an exception for this file we will forget to reenable it once we have linux part implemented. Therefore I add this here so that it can still be include for lint

Reply:
Is the linter picking this up??

This should only happen if the code actually gets built right?

If the linting is causing a problem, perhaps we can somehow tell the linter to skip it, or otherwise we can define the correct type for Linux.

Reply:
I don't think the lint respect the build file
https://github.com/flutter/engine/blob/master/ci/lint.sh

I can't tell what is the correct type for linux until i spent time looks into the linux implementation. I can make this slightly more systematic by define this for linux only instead of this catch all case.

Reply:
SGTM. If we keep ti as a `void*` please file an issue/add a TODO for it.